install_nvm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+NODE_VERSION="v0.10.35" # Latest stable
+
+echo "*********************** Downloading nvm ***********************"
+curl https://raw.githubusercontent.com/creationix/nvm/v0.16.1/install.sh | sh
```

Comment:
I'm a little concerned about the security of nvm. This URL is HTTPS, but some others within the script are not (https://github.com/creationix/nvm/issues/614). It's not a big deal when we're running in Vagrant, but I'd like to be able to use similar techniques for prod.

Two possibilities: we could include a specific version of nvm as a submodule, or we could upgrade to a more recent Vagrant box that has up-to-date Node packages.